packages/netlify-cms/webpack.config.js

```diff
@@ -1,32 +1,66 @@
 const path = require('path');
-const webpack = require('webpack');
 const FriendlyErrorsWebpackPlugin = require('friendly-errors-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
-const pkg = require('./package.json');
-const { plugins } = require('../../scripts/webpack');
-const coreWebpackConfig = require('../netlify-cms-core/webpack.config.js');
+const pkg = require(path.join(process.cwd(), 'package.json'));
```

Comment:
I'd opt for `__dirname` instead of `process.cwd()` here, in case the cli is run from somewhere other than root.

Reply:
`pkg` here should be the specific package, not the root. The logic does not change from the current version.